src/engraving/libmscore/system.h

```diff
@@ -235,6 +235,8 @@ class System final : public EngravingItem
     double squeezableSpace() const;
     bool hasCrossStaffOrModifiedBeams();
 
+    static double yStaffDifference(System* system1, staff_idx_t staffIdx1, System* system2, staff_idx_t staffIdx2);
```

Comment:
Do we really need this function here? I think a better place for it would be libmscore/utils.h (if it will be needed outside of glissando.cpp)